client/src/main/scala/drt/client/components/FlightTableComponents.scala

```diff
@@ -103,21 +103,20 @@ object FlightTableComponents {
     case Some(millis) => SDate(millis).toLocalDateTimeString()
   }
 
-  def timelineCompFunc(flight: Arrival): VdomElement = {
+  def timelineCompFunc: Arrival => TagMod = (flight: Arrival) => {
     Try {
-      timelineFunc(150 - 24, flight.Scheduled, flight.Actual, flight.ActualChox, flight.Estimated, flight.EstimatedChox)
+      timelineFunc(schPct = 150 - 24, sch = flight.Scheduled, act = flight.Actual, actChox = flight.ActualChox, estDt = flight.Estimated, estChoxDt = flight.EstimatedChox)
     }.recover {
       case e =>
-        log.error(s"couldn't render timeline of $flight with $e")
-        val recovery: VdomElement = <.div("uhoh!")
-        recovery
+        log.error(msg = s"couldn't render timeline of $flight with $e")
+        <.div("uhoh!")
```

Comment:
not sure I've ever seen this show up, but we should probably put a better error message in here.

Reply:
This was dead code so i've removed it and a bunch of functions it was using